### PR TITLE
Added References Tab to DiviDevTools Modal

### PR DESCRIPTION
**Pull request by 7ahery**
### Summary
Added new "References" tab to DiviDevTools modal, It extracts all module exports on `window.divi` and lists them with a link to their documentation on https://devalpha.elegantthemes.com.

It also adds a simple search input which lets user to look for exported members.

### CODE REVIEW

File: src/components/references-tree-view/index.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import { map, kebabCase, forEach } from 'lodash';
+import { ReferencesTreeViewProps } from './types';
+import './style.scss';
+
+const ReferencesTreeView = (props: ReferencesTreeViewProps) => {
```
Comment (fikrirasyid) [inline src/components/references-tree-view/index.tsx:6]: Let's add JSDoc here.

**Review by fikrirasyid — CHANGES_REQUESTED** [review]
This looks great! I've tested it and i believe we can tweak it to make it better:

- Let's sort a-z for the package name
- Let's sort a-z for the function name inside the package name
- When search is active and the list is filtered, automatically expand the accordion item
- Accordion item can be expanded, but can't be collapsed. First click expand the accordion content but when you click it again, it didn't collapse the accordion. I think it is more intuitive to collapse it when you click it on expanded state.

**Comment by 7ahery** [comment]
@fikrirasyid I've addressed all the provided feedback, and as an additional improvement, I've added Fuzzy Search support, making it possible to retrieve search results even when there are minor typos or spelling variations in the query.

This improvement improves the user experience by ensuring that users can find relevant information with more ease and accuracy.

**Review by fikrirasyid — DISMISSED** [review]
This looks great. I'm merging this 👍

**Pull request merged**